src/main/java/com/dev/cinema/controllers/CinemaHallController.java

```diff
@@ -0,0 +1,44 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.dto.CinemaHallDto;
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.service.CinemaHallService;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController("/cinemahalls")
+public class CinemaHallController {
+    private CinemaHallService cinemaHallService;
+
+    public CinemaHallController(CinemaHallService cinemaHallService) {
+        this.cinemaHallService = cinemaHallService;
+    }
+
+    @PostMapping("/")
```

Comment:
```suggestion
    @PostMapping("/add")
```

Reply:
implemented Bodya`s naming. Changed as you suggest